repackaging/images.yaml

```diff
@@ -94,4 +94,8 @@ images:
   - name: dynatrace
     type: node
     package: "@dynatrace-oss/dynatrace-mcp-server"
-    version: 0.6.1
\ No newline at end of file
+    version: 0.6.1
+  - name: redis
+    type: python
+    package: git+https://github.com/redis/mcp-redis.git
+    version: main
```

Comment:
Let's not do main. Let's do tagged releases. This also points out that we will need a mechanism to check for new releases, since we aren't pulling this from pypi like other python based mcp servers.

Reply:
I am guessing you did main because they hadn't done the 0.3 release yet. If we get into that situation in the future, use a commit hash rather the main branch.

Reply:
updated